modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2202,6 +2228,32 @@ public void testGetObjectEntriesWithAccountEntryRestricted1()
 			ResourceConstants.SCOPE_COMPANY, String.valueOf(companyId),
 			role.getRoleId(), ActionKeys.VIEW);
 
+		// Regular roles permissions should bypass the account restriction
```

Comment:
Hey @jairmedeiros tests of this class are sorted alphabetically by comments, so to follow the pattern your use case should came before the use above,

Thank you!

Reply:
Done!

Reply:
Hi @MarinhoFeliphe, I removed this test, I noticed the test above is the same haha